amp/WEB-INF/src/org/digijava/module/aim/util/ActivityVersionUtil.java

```diff
@@ -358,9 +358,30 @@ public static AmpActivityVersion initializeActivity(AmpActivityVersion act) thro
         return compareActivities(activityOneId, activityTwoId);
     }
      
-    
-    
+    public static Map<Long, Map<String, List<CompareOutput>>> compareActivities(List<Long> activityId) throws Exception {
```

Comment:
we already have the method that compares to its previous version, we should reuse to avoid code duplicates